README.md

```diff
@@ -55,19 +53,9 @@ yarn add react-native-screens
 expo install react-native-screens
 ```
 
-2.  Enable screens support before any of your navigation screens renders. Add the following code to your main application file (e.g. App.js):
-
-```js
-import { enableScreens } from 'react-native-screens';
-
-enableScreens();
-```
-
-Note that the above code needs to execute before the first render of a navigation screen. You can check the Example's app [App.js](https://github.com/kmagiera/react-native-screens/blob/master/Example/App.js#L16) file as a reference.
-
-3.  Make sure that the version of [react-navigation](https://github.com/react-navigation/react-navigation) you are using is 2.14.0 or higher
+Just make sure that the version of [react-navigation](https://github.com/react-navigation/react-navigation) you are using is 2.14.0 or higher.
 
-4.  You are all set 🎉 – when screens are enabled in your application code react-navigation will automatically use them instead of relying on plain React Native Views.
+You are all set 🎉 – when screens are enabled in your application code react-navigation will automatically use them instead of relying on plain React Native Views.
```

Comment:
Please add here a note with the same note as in other files, with the information how to disable them.

Reply:
Fixed in e6e16cc492349bb0433e3ef952d8afe16fcd6f59